.env.example

```
@@ -0,0 +1,67 @@
+## S3 Bucket Names
+# Logs Bucket. Default name: meeting-baas-logs
+AWS_S3_LOGS_BUCKET=
+# Artifacts Bucket to store MP4, FLAC, Transcription. Default name: meeting-baas-artifacts
+AWS_S3_ARTIFACTS_BUCKET=
+# Audio chunks Bucket to store audio chunks. Default name: Default name: meeting-baas-audio-chunks
+AWS_S3_AUDIO_CHUNKS_BUCKET=
+
+## Media device name overrides
+# Display device. Default: ":99"
+# This needs to be present in the env because Playwright relies on this value to be available in the environment when launching Chrome
+DISPLAY=":99"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)